generator/generate-package-json.ts

```diff
@@ -0,0 +1,34 @@
+import packageJson from './package.json';
```

Comment:
could you just make this a `..` and save yourself a `cp` in `build.sh` line 4?
or am i missing a pitfall here

Reply:
Typescript is being picky and won't let you reference something out of `rootDir`